source/classification_model.py

```diff
@@ -55,43 +58,58 @@ def __len__(self):
         return len(self.image_paths)
 
 
-class ClassificationModel():
-    def __init__(self, num_classes: int = 3, no_epochs=5, **kwargs):
+class ClassificationModel:
+    def __init__(self, num_classes: int = 4, no_epochs=5, **kwargs):
```

Comment:
Why do you change to 4 classes instead of 3?

Reply:
Copy pasted from the issue and did not check :(